test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1

```diff
@@ -3,22 +3,36 @@
 Describe "Get-FormatData" -Tags "CI" {
 
     Context "Check return type of Get-FormatData" {
-
         It "Should return an object[] as the return type" {
             $result = Get-FormatData
-            ,$result | Should -BeOfType "System.Object[]"
+            , $result | Should -BeOfType "System.Object[]"
         }
     }
 
-    It "Can get format data requiring '-PowerShellVersion 5.1'" {
-        $format = Get-FormatData System.IO.FileInfo -PowerShellVersion 5.1
-        $format.TypeNames | Should -HaveCount 2
-        $format.TypeNames[0] | Should -BeExactly "System.IO.DirectoryInfo"
-        $format.TypeNames[1] | Should -BeExactly "System.IO.FileInfo"
-        $format.FormatViewDefinition | Should -HaveCount ($IsWindows ? 4 : 5)
+    Context "Local use: Can get format data requiring v5.1 by default" {
+        $cmds =
+            @{ cmd = { Get-FormatData System.IO.FileInfo } },
+            @{ cmd = { (Get-FormatData System.IO.FileInfo &) | Receive-Job -Wait -AutoRemoveJob } }
+        It "Can get format data requiring v5.1 with <cmd>" -TestCases $cmds {
```

Comment:
```suggestion
        It "Can get format data requiring v5.1 with '<cmd>'" -TestCases $cmds {
```

Reply:
Surprisingly, Pester actually stringifies script blocks _with_ the enclosing `{` and `}`, so you'll get something like:

```none
When remoting, { Get-FormatData System.IO.FileInfo } should return $true for a null-output test
```

So I think that's sufficient - no need to quote as well.